app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="featuredCarousel" class="carousel slide" data-ride="carousel">
```

Comment:
Its usually good practice for anything rendering a collection to have a check up front so that if the collection is empty we don't show any empty user interface elements.

Otherwise we end up with a page looking a little bit like:

<img width="1388" alt="Screen Shot 2020-07-13 at 9 16 48 pm" src="https://user-images.githubusercontent.com/296341/87308715-337a5e80-c54e-11ea-9309-de967d728273.png">

What I would suggest here is wrapping this in a conditional check so that nothing is rendered if there are no features bookies